ext/pdo_mysql/tests/bug53551.phpt

```diff
@@ -4,26 +4,25 @@ Bug #44327 (PDORow::queryString property & numeric offsets / Crash)
 pdo_mysql
 --SKIPIF--
 <?php
-require_once(__DIR__ . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc');
+require_once(__DIR__ . DIRECTORY_SEPARATOR . 'inc' . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc');
 MySQLPDOTest::skip();
-$db = MySQLPDOTest::factory();
 ?>
 --FILE--
 <?php
-include __DIR__ . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc';
+require_once(__DIR__ . DIRECTORY_SEPARATOR . 'inc' . DIRECTORY_SEPARATOR . 'mysql_pdo_test.inc');
 $db = MySQLPDOTest::factory();
 
 $db->setAttribute(PDO::ATTR_EMULATE_PREPARES, 0);
 
-$createSql = "CREATE TABLE `bug53551` (
+$createSql = "CREATE TABLE `test` (
   `count` bigint(20) unsigned NOT NULL DEFAULT '0'
 )";
 
-$db->exec('drop table if exists bug53551');
+$db->exec('drop table if exists test');
 $db->exec($createSql);
-$db->exec("insert into bug53551 set `count` = 1 ");
+$db->exec("insert into test set `count` = 1 ");
 $db->exec("SET sql_mode = 'Traditional'");
-$sql = 'UPDATE bug53551 SET `count` = :count';
+$sql = 'UPDATE test SET `count` = :count';
 $stmt = $db->prepare($sql);
```

Comment:
Please do not do this, each test should use it's own table so we can run them in parallel, and this also goes against #11879